ledger-core/v2/logicrunner/common/conveyor_dispatcher.go

```diff
@@ -108,3 +108,8 @@ func (c *conveyorDispatcher) Process(msg *message.Message) error {
 func NewConveyorDispatcher(conveyor *conveyor.PulseConveyor) dispatcher.Dispatcher {
 	return &conveyorDispatcher{conveyor: conveyor}
 }
+
+type TestApiCall struct {
```

Comment:
imho TestApiCall -> TestAPICall, if golinter is ok
TestAPICall should be moved from here, but i don't know where it should be.